library/test/src/test_result.rs

```diff
@@ -95,8 +90,10 @@ pub fn get_result_from_exit_code(
 ) -> TestResult {
     let result = match status.code() {
         Some(TR_OK) => TestResult::TrOk,
+        // On Windows we use __fastfail to abort, which is documented to use this
+        // exception code.
         #[cfg(windows)]
-        Some(STATUS_ABORTED) => TestResult::TrFailed,
+        Some(const { 0xC0000409u32 as i32 }) => TestResult::TrFailed,
```

Comment:
why not just `const ABORTED_BY_FASTFAIL: i32 = 0xC0000409u32 as i32` like 3 lines up?

Reply:
I like this suggestion and incorporated it into the latest update. At the time I wrote this, I felt more documentation should go into the match arms. After some thought though, I think giving the constant a better name is the right approach. For example, the unix arm uses `libc::SIGABRT` which corresponds to the name of an actual signal.